### PR TITLE
Add _.toPairs, improve _.fromPairs

### LINKED ISSUE #70: fromPairs can be replaced with Object.fromEntries in the modern JS
[mdn docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries)

Support is pretty good too, should we replace it?

### PR DESCRIPTION
**Pull request by 102**
closes #70

### CODE REVIEW

File: src/content/lodash/array/fromPairs/vanilla.js [hunk 1]
```diff
@@ -1,8 +1,2 @@
-const fromPairs = pairs =>
-  pairs.reduce((cache, pair) => {
-    cache[pair[0]] = pair[1]
-    return cache
-  }, {})
-
-module.exports = fromPairs([['a', 1], ['b', 2]])
+module.exports = Object.fromEntries([['a', 1], ['b', 2]])
```
Comment (102) [inline src/content/lodash/array/fromPairs/vanilla.js]: Should we keep previous implementation, or add a comment about `Object.fromEntries` being pretty cutting-edge (e.g. it works only starting from node v12), or MDN link would be enough?

Comment (cedmax) [reply src/content/lodash/array/fromPairs/vanilla.js]: Interesting questions, what do you think?

Comment (102) [reply src/content/lodash/array/fromPairs/vanilla.js]: We can keep both implementations with a clear comment that one way is preferred (due to performance and readability), but has some usage limitations due to being new, and the other one will work almost anywhere

Comment (cedmax) [reply src/content/lodash/array/fromPairs/vanilla.js]: Sounds good to me. Also thank you again for picking up all of these. I was already thinking about adding a contributors section in the readme but with your contributions it has become necessary :)

File: src/content/lodash/array/toPairs/vanilla.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function Foo() {
+  this.a = 1
+  this.b = 2
+}
+
+Foo.prototype.c = 3
+
+module.exports = Object.entries(new Foo())
```
Comment (102) [inline src/content/lodash/array/toPairs/vanilla.js]: `Object.entries` can be used starting from node v6, I don't think it's worth adding pre-`Object.entries` implementation

**Pull request merged**